test/ParallelKernel/test_parallel.jl

```diff
@@ -162,6 +162,7 @@ end
                         return
                     end
                     @parallel write_indices!(A);
+                    if $package == $PKG_AMDGPU AMDGPU.HIP.device_synchronize() end
```

Comment:
HOTFIX: device sync is somehow needed here and in later tests (mostly 3D). One should still investigate why further @omlins

Reply:
This relates to #106 description added in the issue.

Reply:
I will merge it as we need to release today. We can still check this...

Reply:
okay